<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginDialog</class>
 <widget class="QDialog" name="CLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>282</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>302</width>
    <height>282</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>302</width>
    <height>282</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CLoginDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#CLoginDialog{
	border-image: url(:/Resources/background_login.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame{border:1px solid #ddd;border-radius:5px;}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>243</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>35</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lne_username">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
border: none;</string>
           </property>
           <property name="placeholderText">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_passwd">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
border: none;</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>密码</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="tbtn_login">
         <property name="minimumSize">
          <size>
           <width>241</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>241</width>
           <height>41</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Resources/login.png);
color:white;</string>
         </property>
         <property name="text">
          <string>登     录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolButton" name="tbtn_close">
      <property name="geometry">
       <rect>
        <x>264</x>
        <y>13</y>
        <width>27</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>27</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>27</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton#tbtn_close{
	border-image: url(:/Resources/close.png);
}

QToolButton#tbtn_close:hover{
	border-image: url(:/Resources/close_enter.png);
}

QToolButton#tbtn_close:pressed{
	border-image: url(:/Resources/close_pressed.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CSMonitorServer.qrc"/>
 </resources>
 <connections/>
</ui>
